NOTICE.txt

```diff
@@ -1,7 +1,15 @@
-OpenSearch
-Copyright 2021 OpenSearch Contributors
-This product includes software, including Kibana source code, developed by Elasticsearch (http://www.elastic.co).
-Copyright 2012-2021 Elasticsearch B.V.
+OpenSearch (https://opensearch.org/)
+Copyright OpenSearch Contributors
+
+This product includes software, including Kibana source code,
+developed by Elasticsearch (http://www.elastic.co).
+Copyright 2009-2018 Elasticsearch B.V.
```

Comment:
This notice went from 2021 to 2018. Are we sure?

Reply:
revert PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3204